perses/tests/test_repex.py

```diff
@@ -54,16 +54,15 @@ def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
             # Set up repex simulation
             reporter_file = os.path.join(temp_dir, f"{wt_name}-{mutant_name}.nc")
             reporter = MultiStateReporter(reporter_file, checkpoint_interval=10)
```

Comment:
We can set the `checkpoint_interval` to be 100 here. There's no need to checkpoint this frequently.

Reply:
Actually, we don't ever use the positions, so we probably don't need to write to the checkpoint file at all here. 
The energies should be written every iteration regardless of the checkpoint_interval.

Reply:
using 100 here for the `checkpoint_interval` worked okay in the preliminary tests